harness/tests/experiment/utils.py

```diff
@@ -9,6 +10,7 @@
 
 import determined as det
 from determined import core, gpu, keras, workload
+from determined.pytorch import PyTorchTrialController, PyTorchTrialContext, PyTorchTrial, Batch
```

Comment:
introducing a framework dependency into a utils module is a sign that the stuff you are writing doesn't belong in utils, I'm thinking.

Reply:
also, google style guide for imports.